plugins/app/src/main/java/com/mapbox/mapboxsdk/plugins/testapp/activity/BuildingActivity.java

```diff
@@ -0,0 +1,179 @@
+package com.mapbox.mapboxsdk.plugins.testapp.activity;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.SeekBar;
+
+import com.mapbox.androidsdk.plugins.building.BuildingPlugin;
+import com.mapbox.mapboxsdk.constants.Style;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.plugins.testapp.R;
+import com.mapbox.mapboxsdk.style.light.Light;
+import com.mapbox.mapboxsdk.style.light.Position;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import timber.log.Timber;
+
+/**
+ * Activity showcasing building plugin integration
+ */
+public class BuildingActivity extends AppCompatActivity implements OnMapReadyCallback {
+
+  @BindView(R.id.mapView)
+  MapView mapView;
+
+  @BindView(R.id.fabBuilding)
+  View fab;
+
+  @BindView(R.id.seekbarLightRadialCoordinate)
+  SeekBar seekbarRadialCoordinate;
+
+  @BindView(R.id.seekbarLightAzimuthalAngle)
+  SeekBar seekbarAzimuthalAngle;
+
+  @BindView(R.id.seekbarLightPolarAngle)
+  SeekBar seekbarPolarAngle;
+
+  private MapboxMap mapboxMap;
+  private BuildingPlugin buildingPlugin;
+  private boolean isEnabled;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_building);
+    ButterKnife.bind(this);
+    mapView.onCreate(savedInstanceState);
+    mapView.getMapAsync(this);
+  }
+
+  @Override
+  public void onMapReady(MapboxMap mapboxMap) {
+    this.mapboxMap = mapboxMap;
+    buildingPlugin = new BuildingPlugin(mapView, mapboxMap);
+    buildingPlugin.setMinZoomLevel(15);
+    fab.setVisibility(View.VISIBLE);
+    initLightSeekbar();
+  }
+
+  private void initLightSeekbar() {
+    seekbarRadialCoordinate.setMax(24); // unknown?
```

Comment:
I'm not sure what the maximum radius is for a light source, refs https://en.wikipedia.org/wiki/Spherical_coordinate_system